extensions/HuggingFace/typescript/hf.ts

```diff
@@ -178,7 +248,8 @@ export class HuggingFaceTextGenerationModelParser extends aiconfig.Parameterized
     }
 
     if (output.output_type === "execute_result") {
-      return output.data as string;
+      return (output.data as TextGenerationOutput | TextGenerationStreamOutput)
+        .generated_text as string;
```

Comment:
This makes sense to me. I feel we should change the core lib too? Although tbh most code will hit the L242 anyways and never reach here

Reply:
I'm assuming you were referring to similar discussion as here: https://github.com/lastmile-ai/aiconfig/pull/283#discussion_r1399248692

I'll update the python to be similar, sounds like we want to keep the response structure as much as possible